lib/build-view.js

```diff
@@ -158,10 +162,16 @@ export default class BuildView extends View {
     if (0 === w || 0 === h) {
       return;
     }
-
+    const orientation = atom.config.get('build.panelOrientation') || 'Bottom';
+    if ( orientation === 'Left' || orientation === 'Right' ) {
+      this.cache[orientation].width = $('.build .output').width();
+    } else if ( orientation === 'Top' || orientation === 'Bottom' ) {
+      this.cache[orientation].height = $('.build .output').height();
```

Comment:
Without looking any close, I remember there were some issues with the terminal element not "automatically expanding the height" - which I think is how any block DOM element behaves. Maybe [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Using_CSS_flexible_boxes) can be used?